library/x509_crt.c

```diff
@@ -3783,6 +3783,7 @@ int mbedtls_x509_crt_verify_restartable( mbedtls_x509_crt *crt,
     int ret;
     mbedtls_x509_crt_verify_chain ver_chain;
     uint32_t ee_flags;
+    volatile uint32_t flags_fi;
```

Comment:
Same here, should this be initialised to failure value?

Reply:
Ageed.